website/blog/2024-10-23-The-New-Architecture-is-Here.md

```diff
@@ -0,0 +1,390 @@
+---
+title: 'The New Architecture is here'
+authors: [reactteam]
+tags: [announcement]
+date: 2024-10-23
+---
+
+React Native 0.76 with the New Architecture by default is now available on npm!
+
+In the [0.76 release blog post](/blog/2024/10/23/release-0.76-new-architecture) we shared the list of all the significant changes included in this version. In this post, we want to give an overview of the New Architecture and how it is going to shape the future of React Native.
+
+The New Architecture adds support for modern React features including full support for [Suspense](https://react.dev/blog/2022/03/29/react-v18#new-suspense-features), [Transitions](https://react.dev/blog/2022/03/29/react-v18#new-feature-transitions), [automatic batching](https://react.dev/blog/2022/03/29/react-v18#new-feature-automatic-batching), and [`useLayoutEffect`](https://react.dev/reference/react/useLayoutEffect). The New Architecture also includes new [native module](/docs/next/fabric-native-components-introduction) and [native component](/docs/next/fabric-native-components-introduction) systems that enable writing type-safe modules with direct access to native without a bridge.
+
+The release is possible thanks to a ground up rewrite of React Native we’ve been working on since 2018, and we’ve taken special care to make the New Architecture a gradual migration for most apps. In 2021, we created [the New Architecture Working group](https://github.com/reactwg/react-native-new-architecture/) to work with the community to support the New Architecture and ensure a smooth upgrade experience for the entire React ecosystem.
```

Comment:
```suggestion
This release is the result of a ground up rewrite of React Native we’ve been working on since 2018, and we’ve taken special care to make the New Architecture a gradual migration for most apps. In 2021, we created [the New Architecture Working Group](https://github.com/reactwg/react-native-new-architecture/) to collaborate with the community on ensuring a smooth upgrade experience for the entire React ecosystem.
```